app/forms/update_user_password_form.rb

```diff
@@ -0,0 +1,26 @@
+class UpdateUserPasswordForm
+  include ActiveModel::Model
+  include FormPasswordValidator
+
+  attr_accessor :password
+  attr_reader :user
+
+  def persisted?
+    true
+  end
+
+  def initialize(user)
+    @user = user
+  end
+
+  def submit(params)
+    self.password = params[:password]
+
+    return false unless valid?
+    @user.update(params)
+  end
+
+  def mobile_changed?
+    false
+  end
```

Comment:
all of these edit forms (email, mobile, password) are processed by the same controller methods -- https://github.com/18F/identity-idp/pull/347/files#diff-de07531badab1faec6c70713dc76aedcR45